pkgs/development/python-modules/flask-mongoengine/default.nix

```
@@ -15,18 +15,20 @@
 
 buildPythonPackage rec {
   pname = "flask-mongoengine";
-  version = "1.0.0";
+  version = "1.0.0-unstable-2023-11-10";
```

```suggestion
  version = "1.0.0-unstable-2022-08-16";
```
We need to adopt the commit date.

---

Oops. TIL.